### ISSUE TITLE
clearance:install generator does not update existing user class using Rails 5 style models

### ISSUE THREAD

**Issue by cattekin** [thoughtbot/clearance#657]
When running `rails generate clearance:install` on a Rails 5 app, `include Clearance::User` is not injected into an existing User model that was created with the rails generators.
The clearance:install generator is looking for `class User < ActiveRecord::Base`, which has been replaced by `class User < ApplicationRecord`.
Fix would be to also check for the new style as well when injecting `include Clearance`

Not a bug but worth mentioning; the create half of the action will copy in a user model using the old style, which could cause confusion.

generator code for reference:
```ruby
# /lib/generators/clearance/install/install_generator.rb:22
def create_or_inject_clearance_into_user_model
  if File.exist? "app/models/user.rb"
    inject_into_file(
      "app/models/user.rb",
      "  include Clearance::User\n\n",
      after: "class User < ActiveRecord::Base\n"
    )
  else
    copy_file 'user.rb', 'app/models/user.rb'
  end
end
```

**Comment by derekprior** [comment]
Thanks for this. Closing in favor of #663 which is ready for code review now.
